src/prediction_helper.py

```diff
@@ -0,0 +1,197 @@
+# coding=utf-8
+# Copyright 2021 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+"""The helper for prediction."""
+import collections
+import json
+import os
+import pprint
+
+from absl import logging
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+
+from data import retrieval_dataloader
+from tasks import classification
+
+
+def get_recall_at_k_from_dataframe(df, topks=[1, 3, 5, 10]):
```

Comment:
Nit: the default value could be a tuple for immutability.
`topks=(1, 3, 5, 10)`